lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -166,6 +171,36 @@ function distributeExponent(node) {
     ChangeTypes.DISTRIBUTE_EXPONENT, node, newNode, false);
 }
 
+/*
+  Given an expression in the form (base)^exponent, where base is a division node
+  and the exponent is any term, we want to distribute the exponent over each term.
+  Returns a nodeChanged Status object.
+
+  e.g. (x^2/y^2)^(2) -> x^4/y^4
+*/
+function distributeExponentOverDivision(node) {
+  // Base is required to be a division node
+  const base = getContent(node.args[0]);
+  if (!Node.Type.isOperator(base, '/')) {
+    return Node.Status.noChange(node);
+  }
+
+  // Exponent can be any term: polynomial, constant, fraction, etc
+  const outerExponent = getContent(node.args[1]);
+
+  const numerator = base.args[0];
```

Comment:
Ooh so this is a good point and I actually tried it, but then it broke my tests, and I realized we actually want to keep the parens. E.g. `((5x)/6)^2` should go to `(5x)^2/6^2`, not `5x^2/6^2`, so that it's clear the exponent is operating on `5x` and not just `x`.